test/test_gdc_postgres_admin.py

```diff
@@ -109,69 +108,6 @@ def test_create_double(self):
 
         self.engine.execute('SELECT * from node_case')
 
-    def test_create_fails_blocked_without_force(self):
```

Comment:
Yeah, I don't think we need to test whether Postgres DDL takes out a lock.

Can the multiprocessing imports be removed now that these tests are gone?